Makefile.dep

```diff
@@ -12,6 +12,31 @@ ifneq (,$(filter nhdp,$(USEMODULE)))
   USEMODULE += oonf_rfc5444
 endif
 
+ifneq (,$(filter gnrc_netif_default,$(USEMODULE)))
+  USEMODULE += gnrc_netif
+  USEMODULE += netif_default
```

Comment:
(or instead of)

```Makefile
ifneq (,$(filter netif_default gnrc_netif_default,$(USEMODULE)))
    USEMODULE += at86rf231
endif
```

as your fixes do, for that matter.